enterprise/backend/test/metabase_enterprise/metabot_v3/tools/filters_test.clj

```diff
@@ -334,9 +334,9 @@
                              :operation :number-greater-than
                              :value 3}]})))))
     (testing "Missing table results in an error."
-      (is (= {:output (str "No table found with table_id " Integer/MAX_VALUE)}
-             (metabot-v3.tools.filters/filter-records
-              {:data-source {:table-id Integer/MAX_VALUE}}))))))
+      (is (thrown-with-msg? clojure.lang.ExceptionInfo #"Not found."
+                            (metabot-v3.tools.filters/filter-records
+                             {:data-source {:table-id Integer/MAX_VALUE}}))))))
```

Comment:
These are changes in the way errors are reported. This is a consequence of making table and card handling symmetrical.